owncloudApp/src/main/java/com/owncloud/android/files/services/TransferRequester.java

```diff
@@ -216,6 +216,8 @@ private void retry(Context context, Account account, OCUpload upload, boolean re
                 // services and camera uploads feature may try to do it, this is the way to proceed
```

Comment:
This commend could be updated
`Since in Android 0 the apps running in background are not allowed to start background services.
The camera uploads feature may try to do this. A way to solve this is to run the camera upload feature in the foreground.` This way its more clear what happens here.

Reply:
It is not related with current PR, but change applied.